packages/graph/hash_graph/postgres_migrations/V1__initial.sql

```diff
@@ -0,0 +1,270 @@
+CREATE EXTENSION IF NOT EXISTS "btree_gist";
```

Comment:
Where any changes made to the logic?

Reply:
Nope, I took the direct dump from the tool and added some spacing